packages/app/features/top/CreateChatSheet.tsx

```diff
@@ -238,7 +238,7 @@ const CreateChatFormContent = ({
           onScrollChange={(scrolling) => {
             onScrollChange?.(scrolling);
           }}
-          height={400}
+          height={500}
```

Comment:
I don't think this is really what we want. Ideally the `ContactBook` would take up all of the available sheet height minus the button + padding, but I was having a hard time getting all the `flex: 1`'s to properly contain the underlying `SectionList`